payments/templates/bank_account_form.html

```diff
@@ -12,6 +12,20 @@
         <div class="form-group">
             <label for="name">Name on the account</label>
             <input type="text" class="form-control" data-stripe="account_holder_name" autocomplete="off" value="{{ STRIPE_DEBUG.bank.name}}" maxlength="50" />
+            <label for="first_name">First Name</label>
+            <input name="first_name" type="text" class="form-control" autocomplete="off" maxlength="16" value="{{ STRIPE_DEBUG.identity.first_name}}"/>
```

Comment:
If we really want to require these, should we add `required` attribute to all these inputs? It won't validate them server-side, but we're sending these values straight to Stripe anyway, so it seems like the best we can do.

Reply:
Yes, but I'm assuming we'll tackle that as the UX improves.  This was just to pre-empt the initial id validate by Stripe.

Reply:
Hmm ... it's a bit more complicated because the form isn't actually submitted. JS grabs the values and sends to Stripe. We'll have to use `checkValidity()` on the form, but then we'll also have to show error messages ourselves. Merging without changing.